README.md

```diff
@@ -51,3 +51,8 @@ python3 manage.py test directory
 python3 manage.py test directory.filename
 python3 manage.py test directory.filename.TestClassName
 ```
+
+### SwaggerUI
+This seems like something worth adding to the readme:
```

Comment:
Can you take this line out?!?!?!?! It's too meta for me...

Reply:
🔥 🚒